ansible/roles/test/tasks/fdb_mac_expire.yml

```diff
@@ -0,0 +1,68 @@
+- name: clear fdb all
```

Comment:
where do you config the asic mac expire time?

Reply:
@lguohan The asic mac expire timer is in this PR:
https://github.com/Azure/sonic-buildimage/pull/2365

It was set as a default value rather than configurable for now.

Reply:
As Zxu commented, it is default value now, set in PR : Azure/sonic-buildimage#2365

Reply:
can we set the value? It is better to configure the value and adjust the expire timer accordingly.

Reply:
@lguohan : Please let me know if I need to resolve the value part in test case. 
Right now since FDB expiration time is fixed to 300 secs, the test case also sticks to same value.

Reply:
@lguohan 
I set the value from variable {{fdb_aging_time}} to switch.json and then use "swssconfig /etc/swss/config.d/switch.json" to update HW.  I track in this test case that MAC must expire within {{fdb_aging_time}} or 2* {{fdb_aging_time}} time period. 
Kindly review.